ui/jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
I _think_ Jest can use `.ts` for configs, which should be typed in your editor...but I could be wrong.

Reply:
You're right, it does. This was created by their bootstrapping script and I think it just plonks down a .js without seeing what language your project uses. I'll switch 👍